lstmcpipe/config/tests/test_paths_config.py

```diff
@@ -53,73 +54,64 @@ def test_path_config_save():
+           f"/fefs/aswg/data/mc/DL2/20200629_prod5_trans_80/gamma/zenith_20deg/south_pointing/{prod_id}/off0.4deg/dl2_gamma_{prod_id}_test.h5"
     assert cfg.irf_dir("gamma") == \
-           "/fefs/aswg/data/mc/IRF/20200629_prod5_trans_80/zenith_20deg/south_pointing/dummy_prodid/gamma"
+           f"/fefs/aswg/data/mc/IRF/20200629_prod5_trans_80/zenith_20deg/south_pointing/{prod_id}/gamma"
     assert cfg.merge_output_file("gamma", "train") == \
-           "/fefs/aswg/data/mc/DL1/20200629_prod5_trans_80/gamma/zenith_20deg/south_pointing/dummy_prodid/off0.4deg/dl1_gamma_dummy_prodid_train.h5"
+           f"/fefs/aswg/data/mc/DL1/20200629_prod5_trans_80/gamma/zenith_20deg/south_pointing/{prod_id}/off0.4deg/dl1_gamma_{prod_id}_train.h5"
     assert cfg.merge_output_file("gamma", "test") == \
-           "/fefs/aswg/data/mc/DL1/20200629_prod5_trans_80/gamma/zenith_20deg/south_pointing/dummy_prodid/off0.4deg/dl1_gamma_dummy_prodid_test.h5"
+           f"/fefs/aswg/data/mc/DL1/20200629_prod5_trans_80/gamma/zenith_20deg/south_pointing/{prod_id}/off0.4deg/dl1_gamma_{prod_id}_test.h5"
     assert cfg.models_path() == \
-           "/fefs/aswg/data/models/20200629_prod5_trans_80/zenith_20deg/south_pointing/dummy_prodid"
+           f"/fefs/aswg/data/models/20200629_prod5_trans_80/zenith_20deg/south_pointing/{prod_id}"
     assert cfg.r0_dir("gamma") == \
            "/fefs/aswg/data/mc/DL0/20200629_prod5_trans_80/gamma/zenith_20deg/south_pointing/off0.4deg"
     assert cfg.sensitivity_file(offset="0.0deg") == \
-           "/fefs/aswg/data/mc/IRF/20200629_prod5_trans_80/zenith_20deg/south_pointing/dummy_prodid/0.0deg/sensitivity_dummy_prodid_0.0deg.fits.gz"
+           f"/fefs/aswg/data/mc/IRF/20200629_prod5_trans_80/zenith_20deg/south_pointing/{prod_id}/0.0deg/sensitivity_{prod_id}_0.0deg.fits.gz"
     assert cfg.train_dir("gamma") == \
-           "/fefs/aswg/data/mc/DL1/20200629_prod5_trans_80/gamma/zenith_20deg/south_pointing/dummy_prodid/off0.4deg/train"
+           f"/fefs/aswg/data/mc/DL1/20200629_prod5_trans_80/gamma/zenith_20deg/south_pointing/{prod_id}/off0.4deg/train"
     assert cfg.test_dir("gamma") == \
-           "/fefs/aswg/data/mc/DL1/20200629_prod5_trans_80/gamma/zenith_20deg/south_pointing/dummy_prodid/off0.4deg/test"
+           f"/fefs/aswg/data/mc/DL1/20200629_prod5_trans_80/gamma/zenith_20deg/south_pointing/{prod_id}/off0.4deg/test"
 
     for stg in cfg.stages:
 
         prop = getattr(cfg, stg)
-        assert isinstance(prop, list)
+        # assert isinstance(prop, list) or isinstance(prop, dict)
 
         for path in prop:
 
-            assert isinstance(path, dict)
-            if stg != "dl2_to_irfs":  # TODO and merge_dl1 !!
-                assert all(item in ["input", "output"] for item in path)
-            else:
-                assert all(item in ["input", "output", "options"] for item in path)
+            # assert isinstance(path, dict)
```

Comment:
As well as this one, right ? every item in `getattr` should be a dict always (except for `train_pipe`)